### DISCUSSION TITLE
Integrating TTS into Flowise app?

### DISCUSSION THREAD

**Discussion by themeaningofmeaning** [FlowiseAI/Flowise#1612] [Q&A]
How are you guys incorporating TTS into your Flowise projects? (eleven labs, Polly, etc.)

I'm trying to get a sense for the fastest way to prototype.  I'm thinking [Flowise->n8n http request node via curl-->generate audio file via eleven labs --> send audio file and text to external frontend (telegram/discord/others?). 

Anyone have experience with this?

**Comment by HenryHengZJ** [comment]
we're working on the native integration directly into Flowise, stay tuned!

**Comment by remisharrock** [comment]
any news on this ?

**Comment by saxoji** [comment]
ref :  https://github.com/FlowiseAI/Flowise/discussions/2703#discussion-6849685, work well
